docs/zapd_extraction_xml_reference.md

```diff
@@ -155,6 +155,9 @@ u64 gCraterSmokeConeTex[] = {
   - `Width`: Required. Width in pixels of the image.
   - `Height`: Required. Height in pixels of the image.
   - `TlutOffset`: Optional. Specifies the tlut's offset used by this texture. This attribute is only valid if `Format` is either `ci4` or `ci8`, otherwise an exception would be thrown.
+  - `ExternalTlut`: Optional. Specifies that the texture's tlut is in a different file. Takes the filename of the file that contains the tlut.
+  - `ExternalTlutOffset`: Optional. Specifies the offset in the `ExternalTlut` of the tlut for the given texture.
+  - `SplitTlut`: Optional. Specifies that the given texture should take from the upper half of the tlut. Takes a bool.
```

Comment:
```suggestion
  - `SplitTlut`: Optional. Specifies that the given texture should take from the upper half of the tlut. Takes a bool, i.e. one of `true`, `false`, `1`, `0`.
```
May as well make it explicit so people don't have to know the xml spec or read the code. We can always lift it to a Note at the top if we end up with more.